lib/delayed/backend/worker.rb

```diff
@@ -16,7 +16,7 @@ def configure
                       end
         self.delay = config.delay_seconds || 0
         self.timeout = config.visibility_timeout || 5.minutes
-        self.expires_in = config.message_retention_period || 4.days
+        self.expires_in = config.message_retention_period || 345600 # 4 days
```

Comment:
why the change?
and even if justified, why leave the line above get away with that `5.minutes` assignment?

Reply:
FWIW I found that upgrading ActiveRecord to at least `4.2` cures this error, but I did not uncover true cause.